arraycontainer.go

```diff
@@ -439,6 +439,7 @@ func (ac *arrayContainer) lazyorArray(value2 *arrayContainer) container {
 }
 
 func (ac *arrayContainer) and(a container) container {
+	//p("ac.and() called")
```

Comment:
@glycerine 

I am sympathetic to this point of view... 

See http://lemire.me/blog/2016/06/21/i-do-not-use-a-debugger/ 

But I have concerns...

1. Leaving printouts does reduce readability. This seems undeniable.
2. Is this an established best practice? If it were, then we could, at least, rely on a standard for how such printouts should appear...

At a minimum, I think we need to document these printouts. This code gets modified by lots of different people.